src/app/components/atoms/customImage.tsx

```diff
@@ -0,0 +1,15 @@
+import {Image, StyleSheet} from 'react-native';
```

Comment:
this is a generic image component so it must have separate folder/component inside components directory